internal/convert.go

```diff
@@ -203,12 +207,15 @@ type streamingStats struct {
 	DataStreamName                string
 	DataflowJobId                 string
 	DataflowGcloudCmd             string
+	GcsResources                  GcsResources
 	ShardToDataStreamNameMap      map[string]string
 	ShardToDataflowInfoMap        map[string]ShardedDataflowJobResources
 	PubsubCfg                     PubsubCfg
 	ShardToPubsubIdMap            map[string]PubsubCfg
+	ShardToGcsResources           map[string]GcsResources
 	MonitoringDashboard           string
 	ShardToMonitoringDashboardMap map[string]string
+	AggMonitoringDashboard        string
```

Comment:
Similar to `GcsResources`, lets create an object for Dashboard instead of it being a string.